tests/bluetooth/tester/src/gatt.c

```diff
@@ -1697,6 +1697,10 @@ static void discover_complete(struct bt_conn *conn,
 		goto fail;
 	}
 
+	/* always require security validation for current sec level */
```

Comment:
Can we enable this by default when we subscribe? So that user could disable it only if needed.

Reply:
I was thinking about this too, the reason why I didn't make it default was to not change default behavior for existing applications. Although I'm totally fine to enabled this by default if that is a consensus.

Reply:
Actually, thinking about this a bit more I think we can go with enabling this by default.
This is a mitigation against broken or malicious devices and should not break properly behaving applications. I'll update PR.